Signal/src/ViewControllers/Utils/NeverClearView.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+@objc class NeverClearView: UIView {
+    override var backgroundColor: UIColor? {
+        didSet {
+            if backgroundColor?.cgColor.alpha == 0 {
```

Comment:
Outside of the context of this PR, this code is a bit arcane. Could you leave a comment in the code to explain how selecting a cell affects the background color requiring this workaround to avoid recoloring the view?

Otherwise, LGTM.

PTAL @charlesmchen